src/plugin.js

```diff
@@ -179,7 +179,11 @@ export default function ({ types: t }) {
           return;
         }
 
-        const config = localInteropRequire(resolve(process.cwd(), configPath));
+        const config = localInteropRequire(resolve(
+          process.cwd(),
```

Comment:
I think if BABEL_PLUGIN_WEBPACK_LOADERS_CONFIG is set, there is no need in `process.cwd()` here.

Also it's really long long name ;-) 'BABEL_PLUGIN_WEBPACK_LOADERS_CONFIG'

IMHO it's better to allow just pass variables into config string, (it's easy to resolve such string with lodash template for example)